src/open_inwoner/accounts/tests/test_contact_views.py

```diff
@@ -174,24 +127,59 @@ def test_contact_create_send_invite(self):
         body = email.alternatives[0][0]  # html version of the email body
         self.assertIn(invite_url, body)
 
-    def test_multiple_contact_create_without_providing_email(self):
-        ContactFactory(email=None)
+    def test_existing_user_contact(self):
+        existing_user = UserFactory()
+        response = self.app.get(self.create_url, user=self.user)
+        form = response.forms["contact-form"]
+
+        form["first_name"] = existing_user.first_name
+        form["last_name"] = existing_user.last_name
+        form["email"] = existing_user.email
+        response = form.submit(user=self.user)
+        pending_invitation = self.user.contacts_for_approval.first()
+
+        self.assertEqual(response.status_code, 302)
+        self.assertContains(response.follow(), existing_user.first_name)
+        self.assertEqual(existing_user, pending_invitation)
+
+    def test_adding_same_contact_fails(self):
         response = self.app.get(self.create_url, user=self.user)
+        form = response.forms["contact-form"]
+
+        form["first_name"] = self.contact.first_name
+        form["last_name"] = self.contact.last_name
+        form["email"] = self.contact.email
+        response = form.submit(user=self.user)
+        expected_errors = {
+            "__all__": [
+                _(
+                    "Het ingevoerde e-mailadres komt al voor in uw contactpersonen. Pas de gegevens aan en probeer het opnieuw."
+                )
+            ]
+        }
         self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.context["form"].errors, expected_errors)
 
+    def test_email_required(self):
```

Comment:
nice test coverage!
Please add a test when a user tries to create contact with inactive user